extensions/widgets/clock/docs/guide/guide.md

```diff
@@ -0,0 +1,29 @@
+# Clock Widget
+The clock widget displays the current time in a given time zone, using
+an analog clock face.
+
+![Clock widget](images/clock.png)
+
+## Creating a Clock
+A Clock widget can be created by dragging it out from the Tools
```

Comment:
We probably need to decide if we want to capitalise the widget name throughout. I think this is an issue for most of the guides.

Reply:
Good point... It looks like in most of the other guides, title case for section headings is standard. So I think this is one of the few correct cases.